pytorch_toolkit/ote/ote/api.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+
+import yaml
+
+
+def train_args_parser(template_path):
+    parser = argparse.ArgumentParser(formatter_class=argparse.RawDescriptionHelpFormatter)
+    with open(template_path, 'r') as model_definition:
+        config = yaml.safe_load(model_definition)
+
+        parser.add_argument('--train-ann-files', required=True,
+                            help='Comma-separated paths to training annotation files.')
+        parser.add_argument('--train-img-roots', required=True,
```

Comment:
Maybe don't use `img` here (name it `train-data-roots`)? What if the framework is needed at root dir to images and masks...

Reply:
Masks are images in most cases. But I like this point, I will propose to do such interface change.

Reply:
renamed as you proposed